turbolinks.gemspec

```diff
@@ -11,6 +11,7 @@ Gem::Specification.new do |s|
   s.files    = Dir["lib/assets/javascripts/*.js.coffee", "lib/turbolinks.rb", "lib/turbolinks/*.rb", "README.md", "MIT-LICENSE", "test/*"]
   s.test_files = Dir["test/*"]
 
+  s.add_dependency 'rails', '>= 4.0'
```

Comment:
This would put Rails and Turbolinks in an interdependent relationship which I don't know how well works out dependency wise.

I think it's okay to rely on documentation to enforce this. Should people submit bugs on Rails 3.2, we say it's unsupported and close, same as we do on the Rails repo.